### DISCUSSION TITLE
Re-enabling a system rule does nothing

### DISCUSSION THREAD

**Discussion by zerthsky** [evilsocket/opensnitch#778] [Q&A] [answered]
Or maybe I'm doing it wrong. OpenSnitch 1.6.0-rc3, Mint 21. How to reproduce:

1. Locate these 2 rules on the System tab: `[profile-drop-inbound] allow localhost connections` and `[profile-drop-inbound] allow established,related connections`.
2. Disable both.
3. Notice your internet die - no web, no pings. As expected.
4. Enable both rules.
5. Your internet will not restore. Reboot does not help.

The only way to restore the internet is to roll back your system to an earlier snapshot.

**Comment by gustavo-iniguez-goya** [comment]
Thank you @zerthsky , I'll try to reproduce this issue.

**Comment by gustavo-iniguez-goya** [comment]
ok, reproduced. You don't need to roll back your system, open the firewall dialog, and change Inbound policy from Deny to Allow, then again to Deny.

I'll fix the error re-enabling those rules.

**Reply by zerthsky** [reply]
Thank you very much for your prompt response. Your method works but has a curious side effect: the rules in question are duplicated on my system now. [Screenshot](https://postimg.cc/TpkBvQmC)

**Reply by gustavo-iniguez-goya** [reply]
yeah, I'm working on both problems.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
This issue should be fixed with this commit: [afa23f5](https://github.com/evilsocket/opensnitch/commit/afa23f5ac22c6c8eb6b31ed1de12bc7306488db0)

Thank you for reporting this error!

**Reply by zerthsky** [reply]
That's great. Thanks. I'll be waiting for the next RC to test it out.

An unrelated question: how much are you interested in UI feature requests? I have some ideas but nothing important or urgent.

**Reply by gustavo-iniguez-goya** [reply]
I'm always open to new ideas @zerthsky ! Take a look at the exisiting features requests, and if it's not already added, open a new issue.
